x-pack/filebeat/module/cisco/amp/config/config.yml

```diff
@@ -0,0 +1,71 @@
+{{ if eq .input "httpjson" }}
+
+type: httpjson
+config_version: "2"
+interval: {{ .interval }}
+
+auth.basic.user: {{ .client_id }}
+auth.basic.password: {{ .api_key }}
+
+request.url: {{ .url }}
+request.method: GET
+request.timeout: {{ .request_timeout }}
+request.ssl: {{ .ssl | tojson }}
+request.transforms:
+- set:
+    target: url.params.start_date
+    value: '[[.cursor.timestamp]]'
+    default: '[[ formatDate (now (parseDuration "-{{ .first_interval }}")) "2006-01-02T15:04:05-07:00" ]]'
+- set:
+    target: url.params.limit
+    value: 100
```

Comment:
Is this value something that users would like to tweak?

Reply:
I don't see too much value in it, but might be wise to keep it open for users to tweak just in case. It has to exist so we can trigger pagination.